.github/workflows/test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        gradleTask: [unitTest, connectedTest]
```

Comment:
does this mean `azureTest` is being run as part of CI tests now?

Reply:
Yes, exactly